src/core/pal/feature.cpp

```diff
@@ -1240,7 +1240,7 @@ std::unique_ptr< LabelPosition > FeaturePart::curvedPlacementAtOffset( PointSet
 
   LabelInfo *li = mLF->curvedLabelInfo();
 
-  double string_height = li->label_height;
+  double string_height = li->characterHeight;
```

Comment:
Also here, stringHeight? I'm mentioning it because I see you've (nicely) done so in other parts.